### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
Please provide your valuable feedback

### CODE REVIEW

**Review by PuneetP16 — APPROVED** [review]
The code looks good, the design is awesome, just loved it. Ready to merge but if you could provide more context like content or description in the card about the product, it would make more sense of how you are designing it.

**Comment by PuneetP16** [comment]
The layout looks shifted towards the left, if it wasn't done intentionally you could try to centre it to emphasize actual content.

File: component.css [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Nunito&family=Roboto:wght@100;300;400;500;700&display=swap');
-@import url('components\\card\\card.css') ;
+@import url('component\\card\\card.css') ;
+@import url('component\\Badges\\badge.css') ;
+
```
Comment (newsindigital) [inline component.css]: use necessary classes for the HTML, where ever required.

File: component.css [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 :root{
     --black:rgb(20 20 20);
     --red:#e50914;
```
Comment (newsindigital) [inline component.css:10]: better definition for variables. better for searching in variables

File: component.css [hunk 3]
```diff
@@ -18,6 +20,9 @@
 h1,h2,h3,h4,h5,h6{
     color: var(--white);
 }
+p{
+    color: var(--white);
+}
 .small-square-size{
         width: 2.5rem;
```
Comment (newsindigital) [inline component.css]: using bootstrap classes name. Thats good. !!!

**Review by newsindigital — CHANGES_REQUESTED** [review]
1. your theme is black , but in cards background is white due to this your whole text is not visible.

2. theme is good but it can be better :) :)

**Review by manojsarna — APPROVED** [review]
your theme is black , but in cards background is white due to this your whole text is not visible.

theme is good but it can be better :) :)

use necessary classes for the HTML, where ever required.

better definition for variables. better for searching in variables

using bootstrap classes name. Thats good. !!!

File: component.css (hunk 3 above)
Comment (dev-ashishsethi) [reply component.css]: Thanks for the feedback

File: component.css (hunk 2 above)
Comment (dev-ashishsethi) [reply component.css:10]: Thanks for suggestion!

**Pull request merged**